tock/employees/views.py

```diff
@@ -1,16 +1,17 @@
 import datetime
 
+from django.conf import settings
 from django.contrib.auth.models import User
 from django.urls import reverse
-from django.views.generic import ListView, DetailView
+from django.views.generic import DetailView, ListView
 from django.views.generic.edit import FormView
 from rest_framework.permissions import IsAuthenticated
-
-from tock.utils import PermissionMixin, IsSuperUserOrSelf
+from tock.utils import IsSuperUserOrSelf, PermissionMixin
```

Comment:
Just curious  -- and unrelated to this PR --  are you using an opinionated linter (black or whatever)? Should we recommend developing with one?

Reply:
Not beyond the established ones, Flake8 and bandit which are in the docs and CI pipeline. This was me running a sort after adding an import.

That said...I'm not sure if Flake8 is scanning for https://www.python.org/dev/peps/pep-0008/#imports at the moment. Will look into it.